.changeset/sweet-melons-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blobscan/web": minor
+---
+
+feat: Add NavArrows component
```

Comment:
Changelog entries should not follow the commit format; they should be written in past tense.

Can we add something like "Added navigation arrow buttons to navigate to previous and next block"?

Reply:
Done d482a1e